mage_ai/frontend/oracle/components/Table/Cell.tsx

```diff
@@ -86,7 +86,7 @@ function Cell({
     );
   } else if (showProgress) {
     cellEl = (
-      <FlexContainer alignItems={'center'}>
+      <FlexContainer alignItems={'center'} fullHeight fullWidth>
```

Comment:
Can just put "center" without brackets

Reply:
oh I didn't see this. I'll push another one.

Reply:
I'll add this change to the PR when I work on making the progress bar easier to understand what's 100%. I originally only added the fullheight/width because it looked funky on the app even though this branch is focused on the tooltips.